### ISSUE TITLE
Support embed_domain for javascript events

### ISSUE THREAD

**Issue by terrysmith-fiworks** [looker/looker_embed_sso_examples#33]
Please add support for the embed_domain which is required for catching javascript events.

**Comment by dthorpe** [comment]
Hi Terry,

Have you added your host page domain name to the embed domain whitelist on the Admin/Embed page?

-Danny

**Comment by terrysmith-fiworks** [comment]
Yes, I have. I know I'm not the only one having trouble because several people on this page have commented about it not not working with SSO iFrames:
[https://discourse.looker.com/t/javascript-embedded-iframe-events/2298](https://discourse.looker.com/t/javascript-embedded-iframe-events/2298)
I'm not positive on this and have tried it different ways, but I think the problem with the C# code is that it's using Path on the UriBuilder (where the embed_domain seems to go) for both the string that's encoded and for constructing the final path with builder.Uri.
